source/slang/slang-ir-link.cpp

```diff
@@ -1387,8 +1389,21 @@ LinkedIR linkIR(
     //
     insertGlobalValueSymbols(sharedContext, targetProgram->getExistingIRModuleForLayout());
 
-
     auto context = state->getContext();
+
+    // Combine all of the contents of IRGlobalHashedStringLiterals
+    {
+        StringSlicePool pool;
+        IRBuilder& builder = sharedContext->builderStorage;
+        for (IRModule* irModule : irModules)
+        {
+            findGlobalHashedStringLiterals(irModule, pool);
+        }
+        addGlobalHashedStringLiterals(pool, *builder.sharedBuilder);
```

Comment:
Does the linker need to propagate/collect this information to make things work? I would assume that the reflection information is coming from unlinked modules coming from the front-end, and then the output from the linker only affects final code generation (and I'm guessing the global instruction doesn't affect codegen).